README.md

```diff
@@ -307,6 +307,8 @@ deployment:
       - goreleaser
 ```
 
+(Note that if you test multiple versions or multiple OSes you probably want to make sure GoReleaser is just run once)
```

Comment:
maybe remove the parenthesis and prefix the line with `>` (so it become a quote?)